app/views/layouts/application.html.slim

```diff
@@ -3,7 +3,11 @@ html
   head
```

Comment:
You need to copy over the bootstrap starter template and adapt it
http://getbootstrap.com/getting-started/#template